src/databricks/labs/ucx/source_code/linters/context.py

```diff
@@ -83,14 +83,14 @@ def fixer(self, language: Language, diagnostic_code: str) -> Fixer | None:
         if language not in self._fixers:
             return None
         for fixer in self._fixers[language]:
-            if fixer.name == diagnostic_code:
+            if diagnostic_code in fixer.supported_advice_codes:
```

Comment:
Diagnostic or advice?

Reply:
it's the code found in Advice.code so the latter, maybe we need to also rename the parameter